XouDevSpec/Classes/Helper/Extensions.swift

```diff
@@ -28,5 +28,24 @@ public extension UIImageView {
     }
 }
 
+var someView: UIView?
```

Comment:
This doesn't seem like it would work. Does it actually work?

Reply:
Yes, yes it does = D

Reply:
Hmm, ok. Didn't expect it to